### PR TITLE
Run `cargo fmt` once again

### PR DESCRIPTION
**Pull request by fitzgen**
r? @philipc

### CODE REVIEW

**Comment by philipc** [comment]
Which rustfmt version?

**Comment by fitzgen** [comment]
```
$ cargo install rustfmt-nightly
<snip>
$ rustfmt --version
0.2.1-nightly ( )
```

The `rustfmt` crate is an old version of `rustfmt`, and `rustfmt-nightly` is the new thing. Eventually, it will go back to just `rustfmt` but I'm not sure when.

File: examples/dwarfdump.rs [hunk 1]
```diff
@@ -651,19 +701,21 @@ fn dump_exprloc<R: Reader>(data: &gimli::Expression<R>, unit: &Unit<R>) -> Resul
...
             if dwop == gimli::DW_OP_deref_size || dwop == gimli::DW_OP_xderef_size {
                 print!(" {}", size);
             }
         }
-        gimli::Operation::Pick { index } => {
-            if dwop == gimli::DW_OP_pick {
-                print!(" {}", index);
-            }
-        }
+        gimli::Operation::Pick { index } => if dwop == gimli::DW_OP_pick {
```
Comment (philipc) [inline examples/dwarfdump.rs]: Not a blocker, but not keen on this formatting.

Comment (fitzgen) [inline examples/dwarfdump.rs]: I actually prefer it!

/me ducks

**Pull request merged**